packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -195,7 +203,7 @@ const DataGrid = forwardRef((props, ref) => {
   } = props;
 
   const displayedColumns = (hasSelectableRows ? [WorklistDataGridUtils.ROW_SELECTION_COLUMN] : []).concat(pinnedColumns).concat(overflowColumns);
-
+  const tableColumns = tableBodyColumns.length > 0 ? tableBodyColumns : displayedColumns;
```

Comment:
I would expect that we would send data the same way from the WorklistDataGrid and FlowsheetDataGrid.  It seems weird to have two separate implementations in a reusable base asset.

Reply:
Removed